EdFi.Ods.Utilities.Migration/Program.cs

```diff
@@ -4,22 +4,23 @@
 // See the LICENSE and NOTICES files in the project root for more information.
 
 using System;
-using System.Collections.Generic;
-using System.Data.SqlClient;
 using System.IO;
-using System.Linq;
 using System.Reflection;
+using Autofac;
+using Autofac.Extensions.DependencyInjection;
 using CommandLine;
 using EdFi.Ods.Utilities.Migration.Configuration;
-using EdFi.Ods.Utilities.Migration.MigrationManager;
-using EdFi.Ods.Utilities.Migration.Queries;
+using EdFi.Ods.Utilities.Migration.Providers;
 using log4net;
 using log4net.Config;
+using Microsoft.Extensions.DependencyInjection;
 
 namespace EdFi.Ods.Utilities.Migration
 {
     internal class Program
     {
+        private static IContainer _container;
```

Comment:
Rather than introducing a new static variable, just return the container from the builder helper.

Reply:
done